Packs/CommonPlaybooks/Playbooks/playbook-Convert_file_hash_to_corresponding_hashes.yml

```diff
@@ -369,25 +381,386 @@ tasks:
     view: |-
       {
         "position": {
-          "x": 970,
-          "y": 110
+          "x": 1080,
+          "y": 60
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "39":
+    id: "39"
+    taskid: 336d7e69-334b-490c-81c4-4b48788a56be
+    type: condition
+    task:
+      id: 336d7e69-334b-490c-81c4-4b48788a56be
+      version: -1
+      name: Is the hashes retrieved?
+      type: condition
+      iscommand: false
+      brand: ""
+      description: ""
+    nexttasks:
+      '#default#':
+      - "18"
+      "yes":
+      - "40"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: foundIndicators.value
+            iscontext: true
+          right:
+            value: {}
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1350,
+          "y": 540
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "40":
+    id: "40"
+    taskid: 4cbe3c2a-dc3d-46fc-8b6a-b589accfdba3
+    type: regular
+    task:
+      id: 4cbe3c2a-dc3d-46fc-8b6a-b589accfdba3
+      version: -1
+      name: Enrich indicators
+      description: commands.local.cmd.enrich.indicators
+      script: Builtin|||enrichIndicators
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "2"
+    scriptarguments:
+      indicatorsValues:
+        complex:
+          root: inputs.SHA256
```

Comment:
In the previous task, you checked indicators.value, but here you are using inputs.sha256 I would use indicators.value here as well, it makes the playbook more readable